### DISCUSSION TITLE
How do you split this into different files?

### DISCUSSION THREAD

**Discussion by oriongonza** [uutils/coreutils#4481] [Q&A] [answered]
This compiles to a single binary, and then you need to pass the name of the util. If I want to call cat I need to call coreutils cat or create a script for each util I want to use. 

Is there a way to build each util to its own binary?

**Comment by sylvestre** [comment, marked as answer]
Read the fine manual :) https://github.com/uutils/coreutils#gnu-make

**Reply by oriongonza** [reply]
my bad!! I completely missed that in the manual, I only skimmed that part

**Reply by oriongonza** [reply]
how do you build all? pass a glob to cargo?

**Reply by sylvestre** [reply]
yeah, the make knows the list of programs and call cargo the right way
